imagefiles/dockcross

```diff
@@ -146,10 +146,13 @@ FINAL_CONFIG=${ARG_CONFIG-${DOCKCROSS_CONFIG-$DEFAULT_DOCKCROSS_CONFIG}}
 # Set the docker image
 FINAL_IMAGE=${ARG_IMAGE-${DOCKCROSS_IMAGE-$DEFAULT_DOCKCROSS_IMAGE}}
 
-# Handle specicial update command
-[[ "$special_update_command" == "update" ]] && command:update && exit
-[[ "$special_update_command" == "update-script" ]] && command:update-script && exit
-[[ "$special_update_command" == "update-image" ]] && command:update-image && exit
+# Handle special update command
+if [ "$special_update_command" != "" ]; then
+  [[ "$special_update_command" == "update" ]] && command:update
+  [[ "$special_update_command" == "update-script" ]] && command:update-script
+  [[ "$special_update_command" == "update-image" ]] && command:update-image
+  exit $?
```

Comment:
This will return 1 if "update" or "update-script" is passed because $? is the return code of the last command, and the last command here is
```
[[ "$special_update_command" == "update-image" ]] 
```